db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -13,3 +13,63 @@ CREATE TABLE keyed (
 ) ENGINE=InnoDB;
 
 RENAME TABLE temporary_password TO `password`;
+
+--
+-- Rename existing `iplog_history` to `iplog_archive`
+--
+
+RENAME TABLE iplog_history TO iplog_archive;
+
+--
+-- Rename existing `locationlog_history` to `locationlog_archive`
+--
+
+RENAME TABLE locationlog_history TO locationlog_archive;
+
+--
+-- Table structure for new `iplog_history` table
+--
+
+CREATE TABLE iplog_history (
+  mac varchar(17) NOT NULL,
+  ip varchar(45) NOT NULL,
+  start_time datetime NOT NULL,
+  end_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
+) ENGINE=InnoDB;
+
+--
+-- Table structure for table 'iplog'
+--
+
+ALTER TABLE iplog MODIFY mac varchar(17) NOT NULL;
+ALTER TABLE iplog MODIFY ip varchar(45) NOT NULL;
+ALTER TABLE iplog ADD PRIMARY KEY(ip);
+ALTER TABLE iplog DROP INDEX ip_view_open;
+ALTER TABLE iplog DROP INDEX mac_view_open;
+ALTER TABLE iplog DROP INDEX iplog_end_time;
+ALTER TABLE iplog DROP FOREIGN KEY 0_63;
```

Comment:
Can these be turned into a single ALTER TABLE?
That might be faster on larger tables.